app/controllers/sessions_controller.rb

```diff
@@ -6,16 +6,16 @@ def create
     user = User.find_by_email(params[:session][:email])
     if user && user.authenticate(params[:session][:password])
       session[:user_id] = user.id
-      flash[:notice] = "Welcome, #{user.full_name}."
+      flash[:success] = "Welcome, #{user.full_name}."
     else
-      flash[:notice] = 'Invalid login.'
+      flash[:danger] = 'Invalid login.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.